pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -185,22 +185,22 @@ func NewMultiCIDRRangeAllocator(
 	if allocatorParams.ServiceCIDR != nil {
 		ra.filterOutServiceRange(allocatorParams.ServiceCIDR)
 	} else {
-		klog.V(0).Info("No Service CIDR provided. Skipping filtering out service addresses.")
+		klog.InfoS("No Service CIDR provided. Skipping filtering out service addresses")
 	}
 
 	if allocatorParams.SecondaryServiceCIDR != nil {
 		ra.filterOutServiceRange(allocatorParams.SecondaryServiceCIDR)
 	} else {
-		klog.V(0).Info("No Secondary Service CIDR provided. Skipping filtering out secondary service addresses.")
+		klog.InfoS("No Secondary Service CIDR provided. Skipping filtering out secondary service addresses")
 	}
 
 	if nodeList != nil {
 		for _, node := range nodeList.Items {
 			if len(node.Spec.PodCIDRs) == 0 {
-				klog.V(4).Infof("Node %v has no CIDR, ignoring", node.Name)
+				klog.V(4).InfoS("Node has no CIDR, ignoring", "nodeName", node.Name)
```

Comment:
node.Name? I'm not going to comment on the other instances of this -- but we should fix them to be consistent across the component.

Reply:
yeah, I agree `we should fix them to be consistent across the component.`